content/references-rest-api.md

```diff
@@ -61,6 +61,107 @@ Event data
 | channel | true | A single channel name. *Limited to 100 channels.* |
 | socket_id | false | Exclude a connection from the targets of the event. |
 
+## Channels
+
+### GET channel
+
+Channel writes a JSON object to the response with information about the
+requested channel's occupation. 
+
+```
+/apps/<id>/channel/<name>
+```
+
+#####Request
```

Comment:
On the fourth level we should use four `#` and not five. Also add a space between the leading `#` and the text.